### ISSUE TITLE
homepage - Missing first level heading

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#87]
There's no H1 level on the homepage. It seems also that the "25 YEARS of making better yesterdays" doesn't have a heading level attached to it and H1 would probably be appropriate for it.

**Comment by kimisgold** [comment]
This should now be live on the site.
